src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class IsoscelesTrapezoid {
```

Comment:
The class declaration should not start with an empty line. Remove the empty line before the class declaration to adhere to Java style guidelines.